localstack/services/apigateway/apigateway_listener.py

```diff
@@ -228,18 +202,43 @@ def forward_request(self, method, path, data, headers):
             elif integration['type'] == 'AWS_PROXY':
                 if uri.startswith('arn:aws:apigateway:') and ':lambda:path' in uri:
                     func_arn = uri.split(':lambda:path')[1].split('functions/')[1].split('/invocations')[0]
+                    account_id = uri.split(':lambda:path')[1].split(':function:')[0].split(':')[-1]
                     data_str = json.dumps(data) if isinstance(data, dict) else data
 
                     relative_path, query_string_params = extract_query_string_params(path=relative_path)
 
+                    source_ip = headers['X-Forwarded-For'].split(',')[-2]
+
+                    request_context = {
```

Comment:
Perhaps we could add a comment with a link to the docs:
```
# Sample request context:
# https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html#api-gateway-create-api-as-simple-proxy-for-lambda-test
```
Also, can we remove the `None` values from the context? We can add more fields as needed in future iterations of this feature.